tests/untestable/tssl_server.nim

```diff
@@ -0,0 +1,119 @@
+discard """
+  cmd: "nim $target --threads:on -d:ssl $options $file"
+"""
+
+#            Nim - Basic SSL integration tests
+#        (c) Copyright 2020 Nim contributors
+#
+#    See the file "copying.txt", included in this
+#    distribution, for details about the copyright.
+#
+## Runs a simple HTTPS server
+## Used by .github/workflows/ci_ssl.yml
+##
+## Warning: this test performs local networking.
+## Test with:
+## nim c -d:ssl -p:. -d:sync -d:danger -r tests/untestable/tssl_server.nim
+## testssl --ip=127.0.0.1 --csvfile testssl.csv --severity LOW https://127.0.0.1:8333
+
+# testssl --parallel https://localhost:8333
+
+import times
+
+# bogus self-signed certificate
+const
+  reply = "HTTP/1.0 200 OK\r\nServer: Nimtest\r\nContent-type: text/html\r\nContent-Length: 0\r\nStrict-Transport-Security: max-age=63072000; includeSubDomains\r\n\r\n"
+  certFile = "tests/stdlib/thttpclient_ssl_cert.pem"
+  keyFile = "tests/stdlib/thttpclient_ssl_key.pem"
+
+proc log(msg: string) =
+  echo "    [" & $epochTime() & "] " & msg
+
+
+when not defined(windows) and defined(sync):
+
+  from net import Port, newContext
+  import
+    net,
+    openssl
+
+  from ssl_config import CiphersIntermediate
+
+  when defined(multithread):
```

Comment:
ditto, use proper package scoping (also, that defined symbol should be referenced in the test description on top, probably even tested in CI via matrix)